### PR TITLE
Correct security format

### LINKED ISSUE #2217: Auth AADToken and AzureKey do not support at the same time
According to quick start doc
https://github.com/Azure/azure-sdk-for-net/blob/3b2801f96f4f43eccb36f78f3fe4d993af570875/doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md#L209, we should support both AADToken and AzureKey as 
```yaml
security: AADToken;AzureKey
security-header-name: Your-Subscription-Key
security-scopes: https://yourendpoint.azure.com/.default
```
Repro steps:
1. Add above configuration to any service autorest.md (e.g., sdk\metricsadvisor\Azure.AI.MetricsAdvisor\src\autorest.md)
2. Run `dotnet build /t:GenerateCode` to generate code.

Actual result:

> EXEC : fatal   | error : Unexpected security scheme 'AADToken;AzureKey'. Only known schemes are AADToken,AzureKey,Anonymous [D:\GIT\azure-sdk-for-net\sdk\metricsadvisor\Azure.AI.MetricsAdvisor\src\Azure.AI.MetricsAdvisor.csproj]
>   fatal   | Process() cancelled due to failure
> EXEC : error   | error : Plugin modelerfour reported failure. [D:\GIT\azure-sdk-for-net\sdk\metricsadvisor\Azure.AI.MetricsAdvisor\src\Azure.AI.MetricsAdvisor.csproj]

Notice: below two could generate successfully:
```yaml
security: AADToken
security-scopes: https://cognitiveservices.azure.com/.default
```
```yaml
security: AzureKey
security-header-name: Your-Subscription-Key
```

### PR DESCRIPTION
**Pull request by pshao25**
Fix https://github.com/Azure/autorest.csharp/issues/2217

### CODE REVIEW

**Review by lirenhe — APPROVED** [review]

**Review by archerzz — APPROVED** [review]

File: doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md [hunk 1]
```diff
@@ -208,7 +208,9 @@ security-scopes: https://yourendpoint.azure.com/.default
 - Support both credentials(e.g. `-securityHeaderName` and `--securityScope` are specified):
 
 ```yaml
-security: AADToken;AzureKey
+security:
+  - AADToken
```
Comment (chunyu3) [inline doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md:212]: It is need that `-` align with `security`

Comment (pshao25) [reply doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md:212]: How to align? I tried this configuration. It works well.

Comment (chunyu3) [reply doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md:212]: ok.

**Review by chunyu3 — APPROVED** [review]

**Pull request merged**